src/h5web/vis-packs/core/heatmap/HeatmapMesh.tsx

```diff
@@ -114,6 +115,27 @@ const SCALE_SHADER: Record<ScaleType, ScaleShader> = {
       return value >= 0.;
     }`,
   },
+  [ScaleType.Gamma]: {
+    uniforms: (domain: Domain, exponent = 1) => ({
```

Comment:
Don't think the default value is worth it for the exponent here, since it is mandatory in the `ScaleParams` type.

Reply:
It was more to satisfy the type constraint of `ScaleShader`:

```ts
export interface ScaleShader {
  uniforms: (
    domain: Domain,
    exponent?: number
  ) => Record<string, { value: number }>;
  fragment: string;
}
```

I changed it to:
```ts
export interface ScaleShader {
  uniforms:
    | ((domain: Domain) => Record<string, { value: number }>)
    | ((domain: Domain, exponent: number) => Record<string, { value: number }>);
  fragment: string;
}
```